www/js/index.js

```diff
@@ -16,31 +16,32 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-var app = {
+const app = {
```

Comment:
Same with this file. Not sure if I should be editing "www/js/index.js" or if we add it to .eslintignore so ESLint ignores it.